src/main/scala/org/ergoplatform/nodeView/state/UtxoState.scala

```diff
@@ -195,7 +193,7 @@ object UtxoState {
           paranoidChecks = true
         ).get
 
-      assert(persistentProver.digest.sameElements(storage.version.get))
+      require(persistentProver.digest.sameElements(storage.version.get))
```

Comment:
Why not "ensuring"? For post-conditions ensuring is a more natural choice.

Reply:
```def ensuring(cond: A => Boolean): A = { assert(cond(self)); self }```

plus this check is required or just for debug? I think that all our checks are mandatory by default, then we can change them to assert